pom.xml

```diff
@@ -32,6 +32,7 @@
     <module>blueflood-cloudfiles</module>
     <module>blueflood-rollupTools</module>
     <module>blueflood-integration-tests</module>
+    <module>spock-tests</module>
```

Comment:
All other module names begin with `blueflood-`. Is there a good reason to deviate from the standard?

Reply:
DOne